### DISCUSSION TITLE
Server Columns feature like in BetterDiscord.

### DISCUSSION THREAD

**Discussion by ExpertMen** [stoatchat/discussions#387] [Feature Suggestions]
BetterDiscord has Server Columns theme. Basicaly this theme is just can collapses server by default in 3 columns - but you can make more columns, if you want. So, how about Revolt?

**Comment by Rexogamer** [comment]
Can you share a screenshot of this feature?

**Comment by ExpertMen** [comment]
Of course!
[image: image.png]

Your, Expert Plus.


пт, 3 июн. 2022 г. в 19:05, Sophie L ***@***.***>:

**Reply by Rexogamer** [reply]
The image hasn't been sent through properly - can you try sending it directly via GitHub?

**Comment by ExpertMen** [comment]
![image](https://user-images.githubusercontent.com/74980375/171998634-a7de37e6-5fc7-4c9a-9684-1aa99d53c0be.png)
